src/Umbraco.Web.UI.Client/src/views/components/umb-mini-search/umbminisearch.component.js

```diff
@@ -10,7 +10,8 @@
             bindings: {
                 model: "=",
                 onStartTyping: "&?",
```

Comment:
Do we drop `on-start-typing` from this component/directive and get all implementations to use the `on-blur` approach @nielslyngsoe ?

Reply:
No, just for this specific usage of umb-mini-search. So still used in other places.